src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -20,7 +20,7 @@
       "version": "4.0.0-rc2-23907",
       "exclude": "Compile"
     },
-    "MSBuild": "0.1.0-preview-00017",
+    "MSBuild": "0.1.0-preview-00018",
```

Comment:
So should this be closed now or what? MSBuild version is now 0.1.0-preview-00024-160610 in the same file.